apps/docs/content/guides/auth/auth-mfa.mdx

```diff
@@ -568,33 +296,6 @@ If your application uses the Supabase Database, Storage or Edge Functions, just
   className="text-foreground-light mt-8 mb-6 [&>div]:space-y-4"
 >
 
-<AccordionItem
-header={<span className="text-foreground">Why is there a challenge and verify API when challenge does not do much?</span>}
-id="why-is-there-a-challenge-and-verify-api"
->
-
-TOTP is not going to be the only MFA factor Supabase Auth is going to support in the future. By separating out the challenge and verify steps, we're making the library forward compatible with new factors we may add in the future -- such as SMS or WebAuthn. For example, for SMS the `challenge` endpoint would actually send out the SMS with the authentication code. For convenience, you may use `challengeAndVerify` to create and verify a challenge in a single step.
```

Comment:
Moved to TOTP doc. The first question is no longer relevant since we have introduced the phone factor

The second question is incorporated into the docs

Third question moved to TOTP FAQ